src/Tizen.NUI/src/public/Layouting/FlexLayout.cs

```diff
@@ -136,7 +136,15 @@ public static AlignmentType GetFlexAlignmentSelf(View view)
             }
             else
             {
-                return flexAlignmentSelfMap[view];
+                AlignmentType flexAlignmentSelf;
+                if (flexAlignmentSelfMap.TryGetValue(view, out flexAlignmentSelf))
```

Comment:
```suggestion
                if (flexAlignmentSelfMap.TryGetValue(view, out var flexAlignmentSelf))
```

Reply:
Thank you very much for your comment!!
I have updated like your suggestion.